bin/genesis

```diff
@@ -4443,6 +4455,52 @@ EOF
 		line
 		echo "Deploying to BOSH director ${target}"
 		echo
+
+		if [[ ${CI_WAIT_FOR_CLEAN_BOSH} == "true" ]]; then
+			i=1
+			attempts=10
+			wait=10
+			echo "Checking if BOSH has any tasks running... (for up to $(( $attempts * $wait )) seconds)"
+
+			while (( $i <= $attempts ))
+			do
+				bosh target $(spruce json ${DEPLOYMENT_ENV_DIR}/name.yml | jq -r '.name')
+				unset DIRECTOR_URL
+				tasks=$(ask_bosh '/tasks?state=queued,processing,cancelling&verbose=2')
+				if [[ -z "${tasks}" || $? != 0 ]]; then
+					echo "FATAL: Unable to contact BOSH to find existing tasks before deploying. Cowardly refusing to deploy"
+					exit 1
+				fi
+
+				running=$(echo "${tasks}" | jq -r '. | length')
+				if [[ $? != 0 ]]; then
+					echo "FATAL: Unexpected task output from BOSH. Cowardly refusing to deploy"
+					echo "Unexpected response:"
+					echo
+					cat << EOF
+$tasks
+EOF
```

Comment:
I didn't want to worry about any special characters (like newlines) that may have been inside $tasks not being printed/escaped properly through just an `echo` call.